tests/integrationv2/tox.ini

```diff
@@ -9,9 +9,9 @@ passenv = LD_LIBRARY_PATH OQS_OPENSSL_1_1_1_INSTALL_DIR
 deps =
     pep8
     pytest==5.3.5
-    pytest-xdist
+    pytest-xdist==1.34.0
 commands =
-    pytest -n2 --cache-clear -rpfsq \
+    pytest -n auto --cache-clear -rpfsq \
```

Comment:
How was performance with `auto`? I don't think we would see negative impact in the environments where we run. But there is definite decrease with too many cores (8 started to break things for me).

Reply:
8 cores appeared to be ok on CodeBuild; 72 certainly broke it though.

Reply:
ok, reverting this- after repeated test runs, 8 is occasionally failing.

Reply:
:/ This may be caused by the port distribution lock not working properly, but a problem for another time.